code-challenges.test.js

```diff
@@ -52,5 +117,36 @@ var numbersToAdd2 = [0, 7, -8, 12]
 var numbersToAdd3 = []
 // Expected output: []
 
+/**
+ * - If the array is empty, return an empty array
+ * - Loop the entire array
+ * - Add the accumulative sum to a variable called "sumList"
+ * - Return sumList
+ */
+
+const accumulatedSum = (arr) => {
+  if (arr.length === 0) return [];
+
+  const sumList = [];
+  let j = 0;
+
+  for (let i = 0; i < arr.length; i++) {
+    j += arr[i]
+    sumList.push(j);
+  }
+  return sumList;
+};
```

Comment:
With the logic you have set up here I don't think you need the if statement. If you pass an empty array to the for loop it should return an empty array.